assets/js/state/lastExecutions.js

```diff
@@ -5,6 +5,7 @@ const initialState = {};
 const initialExecutionState = {
   data: null,
   loading: false,
+  executionStarted: false,
```

Comment:
Couldn't we use the current `status` field instead of creating a new one?

Reply:
done